xrpl/models/transactions/payment_channel_claim.py

```diff
@@ -104,3 +111,12 @@ class PaymentChannelClaim(Transaction):
         default=TransactionType.PAYMENT_CHANNEL_CLAIM,
         init=False,
     )
+
+    credential_ids: Optional[List[str]] = None
+    """Credentials associated with sender of this transaction. The credentials included
+    must not be expired."""
+
+    def _get_errors(self: Self) -> Dict[str, str]:
+        errors = super()._get_errors()
+
+        return errors | validate_credential_ids(self.credential_ids)
```

Comment:
Same here

Reply:
addressed in [8e496e1](https://github.com/XRPLF/xrpl-py/pull/759/commits/8e496e18ddddb58ac27457210e10716cdae80eeb)